ursula_cli/shell.py

```diff
@@ -34,6 +34,23 @@
 MINIMUM_ANSIBLE_VERSION = '1.9'
 
 
+def which(program):
+    def is_exe(x): return os.path.isfile(x) and os.access(x, os.X_OK)
```

Comment:
one-lining this is AFAIK a violation of pep8, so I'd just go back to the old way.